### ISSUE TITLE
Rule Proposal: use-v-on-exact

### ISSUE THREAD

**Issue by mysticatea** [vuejs/eslint-plugin-vue#271]
From #269.

**Please describe what the rule should do:**

```html
<button @click="select" @click.ctrl="toggle">test</button>
```

In that case, I think that people expected <kbd>click</kbd> triggers the `select` and <kbd>ctrl</kbd>+<kbd>click</kbd> triggers the `toggle`. But actually, the `select` is triggered by the both cases.

This rule reports `v-on` directives if multiple `v-on` directives exist and those have different system modifiers.

    Consider to use '.exact' modifier.

**What category of rule is this? (place an "X" next to just one item)**

[ ] Enforces code style
[X] Warns about a potential error
[X] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```html
<!-- ✓ GOOD -->
<button @click="foo"/>
<button @click.ctrl="foo"/>

<!-- ✘ BAD -->
<button @click="foo" @click.ctrl="foo"/>
```

**Comment by michalsnik** [comment]
I like this proposition. Looks like a good `heads up` rule :)

**Comment by chrisvfritz** [comment]
I like this rule too! I'm thinking it could even go in `essentials` and is a candidate for the style guide. @michalsnik Anything you could use from me to make development on this easier?

**Comment by michalsnik** [comment]
I think all is clear here and ready to be implemented!

**Comment by armano2** [comment]
i can make it but i have question: should this work only for build in events or for all?
